### ISSUE TITLE
Reproducible timestamps

### ISSUE THREAD

**Issue by ebourg** [wvengen/proguard-maven-plugin#279]
The entries in the jar file generated have the date set to the build date. It should be set to the timestamp specified by the `project.build.outputTimestamp` property in the POM to make the build reproducible.

https://maven.apache.org/guides/mini/guide-reproducible-builds.html

**Comment by lasselindqvist** [comment]
As long as Proguard itself does not support such an option (https://www.guardsquare.com/manual/configuration/usage), the Maven plugin cannot tell it to use a constant timestamp

Maybe there is another plugin that would "rewrite" the obfuscated jar and produce the result you need.

**Comment by ebourg** [comment]
> Maybe there is another plugin that would "rewrite" the obfuscated jar and produce the result you need.

Indeed, [reproducible-build-maven-plugin](https://zlika.github.io/reproducible-build-maven-plugin) can do this.
